app/uk/gov/hmrc/incorporatedentityidentification/controllers/RegisterBusinessEntityController.scala

```diff
@@ -32,12 +32,30 @@ class RegisterBusinessEntityController @Inject()(cc: ControllerComponents,
                                                  val authConnector: AuthConnector
                                                 )(implicit ec: ExecutionContext) extends BackendController(cc) with AuthorisedFunctions {
 
-  def register(): Action[JsValue] = Action.async(parse.json) {
+  def registerCompany(): Action[JsValue] = Action.async(parse.json) {
     implicit request =>
       authorised() {
         val crn = (request.body \ "company" \ "crn").as[String]
         val ctutr = (request.body \ "company" \ "ctutr").as[String]
-        registerWithMultipleIdentifiersService.register(crn, ctutr).map {
+        registerWithMultipleIdentifiersService.registerCompany(crn, ctutr).map {
+          case RegisterWithMultipleIdentifiersSuccess(safeId) =>
+            Ok(Json.obj(
+              "registration" -> Json.obj(
+                "registrationStatus" -> "REGISTERED",
+                "registeredBusinessPartnerId" -> safeId)))
+          case RegisterWithMultipleIdentifiersFailure(status, body) =>
+            Ok(Json.obj(
+              "registration" -> Json.obj(
+                "registrationStatus" -> "REGISTRATION_FAILED")))
+        }
+      }
+  }
+  def registerRegisteredSociety(): Action[JsValue] = Action.async(parse.json) {
+    implicit request =>
+      authorised() {
+        val crn = (request.body \ "registeredSociety" \ "crn").as[String]
+        val ctutr = (request.body \ "registeredSociety" \ "ctutr").as[String]
```

Comment:
```suggestion
        val crn = (request.body  \ "crn").as[String]
        val ctutr = (request.body \ "ctutr").as[String]
```

These are going to be the fields that are sent for both register APIs going forward